### PR TITLE
netstandard1.3;netstandard2.0;net40;net35

### PR DESCRIPTION
**Pull request by maikebing**

### CODE REVIEW

File: src/LiteQueue/LiteQueue.cs [hunk 1]
```diff
@@ -50,8 +50,18 @@ public LiteQueue(LiteCollection<QueueEntry<T>> collection, bool transactional =
             _collection = collection;
             _transactional = transactional;
             _collection.EnsureIndex(x => x.IsCheckedOut);
-        }
-
+        }
+        /// <summary>
+        /// Creates a collection for you in the database, collection's name is  Type name
```
Comment (nbushby) [inline src/LiteQueue/LiteQueue.cs]:
```suggestion
        /// Creates a collection for you in the database, collection's name is <typeparamref name="T"/>
```

Comment (nbushby) [reply src/LiteQueue/LiteQueue.cs]: See: https://docs.microsoft.com/en-us/dotnet/csharp/programming-guide/xmldoc/typeparam

File: src/LiteQueue/LiteQueue.csproj [hunk 2]
```diff
@@ -11,11 +11,11 @@
     <RepositoryType>Git</RepositoryType>
     <PackageTags>LiteDB, MSMQ, Queue</PackageTags>
     <PackageLicenseUrl>https://github.com/NomadeonSoftwareLLC/LiteQueue/blob/master/LICENSE</PackageLicenseUrl>
-    <Version>1.0.1</Version>
+    <Version>1.0.2</Version>
```
Comment (nbushby) [inline src/LiteQueue/LiteQueue.csproj]:
```suggestion
    <Version>1.1.0</Version>
```

Comment (nbushby) [reply src/LiteQueue/LiteQueue.csproj]: Using pattern:
MajorFeature.MinorFeature.Bugfix

**Review by nbushby — COMMENTED** [review]
You can also update LiteQueue.nuspec file with new version

**Comment by nbushby** [comment]
When I review the diff on LiteQueue.cs, it makes it look like every line has changed (270 additions, 259 deletions). Did the line endings or spacing get messed up?

**Comment by maikebing** [comment]
Yes, so you need hide whitespace changes！I'm sorry ！

**Review by nbushby — APPROVED** [review]
Looks good when I hide whitespace changes. Thank you.

**Pull request merged**